Sources/MapboxCoreNavigation/PredictiveCacheManager.swift

```diff
@@ -44,7 +44,10 @@ public class PredictiveCacheManager {
                                                  maxConcurrentRequests: UInt32 = 2) -> PredictiveCacheController? {
         let cacheOptions = PredictiveCacheControllerOptions(version: version,
                                                             dataset: dataset,
-                                                            concurrency: maxConcurrentRequests)
+                                                            // TODO: Make sure to use correct domain.
+                                                            dataDomain: .navigation,
```

Comment:
@Udumft, there are two domains: `maps` and `navigation`, it'd be good if you can check which one is correct.

Reply:
`.maps` is the correct one for this case: `CacheOptions` are configured for handling Maps caches, while `Navigator`'s controller is initialized without additional options and automatically detects navigator settings.